### PR TITLE
Add description to ConfigureHttpClient function

### PR DESCRIPTION
**Pull request by mgold1234**
# Description

FIXES: 

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [x] Refactor

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -58,3 +58,5 @@ go/pkg/mod/*
 /build/
 /coverage.json
 /coverage.html
+
+rebase.sh
```
Comment (loadtheaccumulator) [inline .gitignore:62]: Please remove this rebase.sh script.

File: rebase.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (loadtheaccumulator) [inline rebase.sh:1]: Please remove this file.

File: pkg/clients/client.go [hunk 3]
```diff
@@ -34,6 +34,8 @@ func GetOutgoingHeaders(ctx context.Context) map[string]string {
 	return headers
 }
 
+// ConfigureHttpClient return https client in case there is TLS CA path,
```
Comment (loadtheaccumulator) [inline pkg/clients/client.go:37]: I would reword to say it adds a TLS config to an http client.
This function is called on the assumption there is a TLS CA Path
Line below looks awesome, but you could mention that if there is no TLS CA, it logs an error and returns the original unmodified client.

**Review by loadtheaccumulator — CHANGES_REQUESTED** [review]

File: rebase.sh
Comment (ldjebran) [inline rebase.sh:1]: _New github feature , to comment on the whole file :smiley:_ 

Please remove this file.

**Comment by loadtheaccumulator** [comment]
It looks like this grew in scope from just adding a description. Please separate them so that a problem with one does not interfere with the other.

File: pkg/clients/client.go (hunk 3 above)
Comment (mgold1234) [reply pkg/clients/client.go:37]: @loadtheaccumulator added the changes

**Review by loadtheaccumulator — APPROVED** [review]
lgtm

**Review by ldjebran — APPROVED** [review]
